add celery_working_process_count and celery_idle_process_count metrics

ori-panjaya opened the pull request.

orihasse committed:
add celery_working_process_count and celery_idle_process_count metrics

tamirzheleznyak commented on src/exporter.py:
```diff
@@ -132,6 +133,19 @@ def __init__(
             registry=self.registry,
         )
+        self.celery_working_process_count = Gauge(
+            f"{metric_prefix}working_process_count",
```
Will there be an underscore between metric_prefix and working_process_count?

ori-panjaya replied:
metric_prefix="celery_"

tamirzheleznyak committed:
Mannuel DEPLOY tutorial

tamirzheleznyak committed:
feat: allow zero queues metrics in celery (#2)

tamirzheleznyak committed:
refactor: async background broker collection + LLEN pipeline (#3)

Three changes that together stop /metrics from ever blocking on the
broker, fixing the slow-scrape failure mode that caused the
2026-05-06 prod incident (Prometheus repeatedly timing out trying to
scrape celery-exporter, KEDA blind, queue-length alerts silently
no-op'd against missing series).
